HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponentsController.swift

```diff
@@ -107,7 +107,9 @@ public final class PaymentComponentsController: BottomSheetsProviderProtocol, Gi
     var documentId: String?
     // Errors stack received from API. We will show them for the clients
     var errors: [String] = []
-
+    
+    // Store Share Bottom Sheet for dismissed native share modal
```

Comment:
[clarification] What is this about? Is not very clear for me 😶‍🌫️ .

Reply:
It is an edge case that needs to be handled when the user jumps to the native share pdf and taps outside. The request was to show the previous share onboarding bottom sheet (the one with the QR)